sink/event_bridge.go

```diff
@@ -0,0 +1,178 @@
+package sink
+
+import (
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/eventbridge"
+	log "github.com/sirupsen/logrus"
+)
+
+// EventBridge ...
+type EBSink struct {
+	session     *session.Session
+	eventbridge *eventbridge.EventBridge
+	busName     string
+	stopCh      chan interface{}
+	putCh       chan []byte
+	batchCh     chan [][]byte
+}
+
+// New Event Bus ...
+func NewEventBus() (*EBSink, error) {
+	busName := os.Getenv("SINK_EVENT_BUS_NAME")
+
+	if busName == "" {
+		return nil, fmt.Errorf("[sink/eventbridge] Missing SINK_EVENT_BUS_NAME")
+	}
+
+	sess := session.Must(session.NewSession())
+	svc := eventbridge.New(sess)
+
+	req, resp := svc.DescribeEventBusRequest(&eventbridge.DescribeEventBusInput{
+		Name: aws.String(busName),
+	})
+
+	fmt.Println(resp)
```

Comment:
should be removed or converted to a `log.Debugf`

Reply:
converted